module/common/SystemInfo.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file SystemInfo.h
+ * @brief 走行システムで統一する情報をまとめたファイル
+ * @author takahashitom
+ */
+
+#ifndef SYSTEM_INFO_H
+#define SYSTEM_INFO_H
+
+// ロボコンの部屋で取得した輝度
+static constexpr int BLACK_BRIGHTNESS = 3;   // 黒の輝度
+static constexpr int WHITE_BRIGHTNESS = 93;  // 白の輝度
+
+static constexpr double RADIUS = 50.0;  // 車輪の半径[mm]
+static constexpr double TREAD = 125.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]
+
+static constexpr char RAS_PI_IP[16] = "172.20.1.1";
+
+static constexpr int ANGLE_SERVER_PORT = 10338;  // 角度算出用サーバのポート番号
```

Comment:
おそらくRotation.hでincludeしてるから、このファイル追加したと思うんだけど、現状使ってないものは削除するか、コメントを残して欲しい。削除したほうがスッキリして良いかも。
ちなみに残すのは現状以下。
```
static constexpr double RADIUS = 50.0;  // 車輪の半径[mm]
static constexpr double TREAD = 125.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]
```
そして、Mileage.cppにも```static constexpr double RADIUS = 50.0;  // 車輪の半径[mm]```があるので、それを消して、```#include "SystemInfo.h"```を追加すれば、SystemInfo.hにまとめている走行システムで統一する情報を上手く使った実装になると思います。

下は雑だけどコメントの例です。
```suggestion
// 現状未使用
// ロボコンの部屋で取得した輝度
static constexpr int BLACK_BRIGHTNESS = 3;   // 黒の輝度
static constexpr int WHITE_BRIGHTNESS = 93;  // 白の輝度

static constexpr double RADIUS = 50.0;  // 車輪の半径[mm]
static constexpr double TREAD = 125.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]

// 現状未使用
static constexpr char RAS_PI_IP[16] = "172.20.1.1";

// 現状未使用
static constexpr int ANGLE_SERVER_PORT = 10338;  // 角度算出用サーバのポート番号
```

Reply:
レビューありがとうございます。
自分も削除した方がスッキリしていいと思ったので、現状つかっていない情報は削除しました。
使用する情報が増えた場合は、その時追加していく形でお願いします。
c700fe0bc5efafd7b5f9f168985bf3a8bf5c3c1b